src/crwbmg.c

```diff
@@ -152,7 +152,7 @@ void cobfl( char *bfl, char io )
 }
 
 /**
- * This function reads the next BUFR message from the system file that was opened via the most
+ * Read the next BUFR message from the system file that was opened via the most
```

Comment:
Let's change "system file" to just "file".

Reply:
OK, good catch.

Reply:
I'll go ahead now and remove all of the "system file" references throughout the library to be just "file".